CHANGELOG.md

```diff
@@ -1,6 +1,43 @@
 # Changelog
 
 ## 1.5.0 (Unreleased)
+### Breaking Change Warning
+In accordance with our [versioning policy](https://github.com/corretto/amazon-corretto-crypto-provider/blob/master/VERSIONING.rst),
+we post warnings of upcoming changes which may cause compatibility issues.
+As always, we expect that these changes will not impact the vast majority of consumers and can be picked up automatically provided you have good unit and integration changes.
+
+Starting in ACCP vesion 1.6.0, EC key pair generation will throw an `InvalidParameterException` if initialized to a keysize not on the following list.
+For these explicit sizes (only), the behavior will remain unchanged with ACCP selecting the corresponding "secp*r1" curve
+(which, in these cases, is also the corresponding NIST Prime Curve).
+
+**Supported keysize values:**
+* 192
+* 224
+* 256
+* 384
+* 521
+
+This means that the following code will start failing.
```

Comment:
```suggestion
This means that the following code will start failing.
```
Therefore, the following code fails because it specifies an unsupported value.

Do we really need all of this explanation? These are the supported values. Everything else fails. Do we need a code sample for that?